.github/workflows/ci.yaml

```
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc-ver: ["9.4.5", "9.6.2", "9.8.2"]
+        ghc-ver: ["9.6.2", "9.8.2", "9.10.1"]
```

Use the latest minor releases?

```suggestion
        ghc-ver: ["9.6.6", "9.8.4", "9.10.1"]
```

---

Let's do this separately as part of #11